wesnoth-multiplayer.po

```diff
@@ -891,7 +879,7 @@ msgstr ""
 #. [side]
 #: data/multiplayer/scenarios/2p_Caves_of_the_Basilisk.cfg:170
 msgid "The Spider"
-msgstr ""
+msgstr "Pavouci"
```

Comment:
všiml jsem si, že jste změnil Pavouk na Pavouci

Na mapě Baziliškova sluje (asi by mělo být Baziliškovy) jsou sochy. Jedna z nich je (jednoho) pavouka. Proč by se měla jmenovat pavouci?
```suggestion
msgstr "Pavouk"
```

Reply:
Má to označení side. Takže předpoládám, že je to stejně jako u jiných řetězců označení strany. Označení stran je všude přeloženo v množném čísle, i v případě, že se v originále jedná o číslo jednotné. Takže jsem to radši takhle upravil.
Je možné, že se mýlím. Nebo že by to chtělo celé nějak upravit.

Reply:
Tak koukám, že side asi není vždy označení strany, resp. někdy je to označení hráče, někdy frakce a někdy strany (více spojených hráčů).